content/components/segmented-control.mdx

```diff
@@ -0,0 +1,426 @@
+
+## Content
+
+### Buttons
+
+A segmented control should have 2–5 choices with text labels, or up to 6 icon-only buttons. Too many choices can be difficult to parse and take up too much horizontal space. If there are more than 5 choices, use another choice selection pattern such as a group of radio inputs or a dropdown menu.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180623808-99d9376f-73bd-4c1c-a9e1-495c84818276.png"
+      role="presentation"
+    />
+    <Caption>Use a segmented control for picking from 5 (6 for icon-only buttons) or fewer options.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180623811-e503fc57-4220-46c3-b40e-23c38c9a74c5.png"
+      role="presentation"
+    />
+    <Caption>Don't use a segmented control for more than 5 options.</Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180623815-d72a2c84-8353-4b84-965a-abe6ce1da724.png"
+      role="presentation"
+    />
+    <Caption>
+      Use a segmented control for selecting from a list of static values. You may use an auxiliary dropdown adjacent to
+      the segmented control.
+    </Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624829-d10c1324-378c-435c-a39b-c41b1ffb8081.png"
+      role="presentation"
+    />
+    <Caption>Don't use a segmented control with a choice that is a dropdown.</Caption>
+  </Dont>
+</DoDontContainer>
+
+### Button icons
+
+<Box
+  display="flex"
+  alignItems="flex-start"
+  flexDirection={['column', 'column', 'column', 'column', 'row']}
+  sx={{gap: 3}}
+>
+  <Text as="p" mt="0">
+    An icon can be rendered in a segmented control button as a visual cue to help users understand the choice that
+    button represents.
+  </Text>
+  <img
+    src="https://user-images.githubusercontent.com/2313998/180623824-41ac7a02-0895-40f7-a973-0aa055a9503c.png"
+    role="presentation"
+    width="456"
+  />
+</Box>
+
+#### Effective visual cues
+
+Be consistent with the way visual cues are used in each button of the segmented control. Use either all icons, all text labels, or all text labels with icons.
+
+Do not change icons to communicate state.
+
+<DoDontContainer>
+  <Do>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624836-920ff9c3-ba88-4137-8d9d-6f4d06271e08.png"
+      role="presentation"
+    />
+    <Caption>Use icons to as a visual cue to communicate what the choice does.</Caption>
+  </Do>
+  <Dont>
+    <img
+      src="https://user-images.githubusercontent.com/2313998/180624838-1915681f-1c50-4d0e-94e9-98f3537cf285.png"
+      role="presentation"
+    />
+    <Caption>Change or add icons to indicate whether the choice is selected.</Caption>
+  </Dont>
+</DoDontContainer>
+
+<DoDontContainer>
+  <Do>
+    <img
```

Comment:
The half-sized images are looking blurry on non-high-resolution monitors. Preview:

![Screen Shot 2022-08-01 at 11 09 20](https://user-images.githubusercontent.com/293280/182213973-72f1d806-3984-45c7-a995-b50b213d0514.png)

I've documented in this [Figma link](https://www.figma.com/file/HnxaGzO6HrEe5dgx6m5BsH/%5BArchived%5D-Overlay-patterns-(FY22-Q4)?node-id=3%3A47278) which sizes we should use for documentation images. @mperrotti let me know if you have any questions!! 💖